### PR TITLE
Redesigned docs to resemble shadcn docs

### PR DESCRIPTION
**Pull request by hqasmei**
I ran prettier formatter as well, heads up

### CODE REVIEW

**Comment by danstepanov** [comment]
This looks a lot better! Can you make the github link in the navbar go to "https://github.com/danstepanov/create-expo-stack"

**Comment by hqasmei** [comment]
Just updated the Github Link

File: docs/src/components/Header/ModeToggle.tsx [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+  DropdownMenu,
+  DropdownMenuContent,
+  DropdownMenuItem,
+  DropdownMenuTrigger,
+} from "@/components/ui/dropdown-menu";
+
+export function ModeToggle() {
+  const [theme, setThemeState] = React.useState<
+    "theme-light" | "dark" | "system"
+  >("theme-light");
```
Comment (danstepanov) [inline docs/src/components/Header/ModeToggle.tsx:15]:
```suggestion
  >("system");
```

**Review by danstepanov — CHANGES_REQUESTED** [review]
Just one nit and we can merge

**Comment by danstepanov** [comment]
@hqasmei Can you try spending no more than like 30 min to try to get it to change based on the system config? It currently is just set to dark unless you manually toggle.

https://github.com/danstepanov/create-expo-stack/assets/5482800/f5889280-db0e-4980-92ac-4c36b73f22a5

**Comment by danstepanov** [comment]
If it isn't clear within 30 then fuck it, ship it.

**Review by danstepanov — APPROVED** [review]
LGTM

**Pull request merged**